lib/Corres_UL.thy

```diff
@@ -903,7 +903,13 @@ lemma corres_returnOk:
   apply wp
   done
 
-lemmas corres_returnOkTT = corres_trivial [OF corres_returnOk]
+lemma corres_returnOkTT:
+  "r x y \<Longrightarrow> corres_underlying sr nf nf' (r' \<oplus> r) \<top> \<top> (returnOk x) (returnOk y)"
```

Comment:
did something change here, or are you writing it out?

Reply:
It changed slightly, the `lemmas` version had some state relation junk left over in it that didn't do anything. It didn't really hurt either, but it was a bit messy.